### DISCUSSION TITLE
Autonumber not working [May 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2195] [Discord Threads]
> 🙋‍♂️ **francol10**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### please create a issue in github  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/976867069985431583)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
